packages/pagination/stories/2-Pagination.stories.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { Pagination } from '@zendeskgarden/react-pagination';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Pagination/Pagination',
+  component: Pagination
+} as Meta;
+
+export const Default: Story = ({ currentPage, totalPages, pagePadding, pageGap }) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <Pagination
+          totalPages={totalPages}
+          pagePadding={pagePadding}
+          pageGap={pageGap}
+          currentPage={currentPage}
```

Comment:
I don't think we're going to want to see `currentPage` controlled via knobs in this example. Maybe `Initial currentPage`? Otherwise, the example should control `currentPage` via `onChange` as seen with the current example: https://zendeskgarden.github.io/react-components/pagination/#basic